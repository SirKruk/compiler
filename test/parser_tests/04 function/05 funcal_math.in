(2+3) * f((4+5))
