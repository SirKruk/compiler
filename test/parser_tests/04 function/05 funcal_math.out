<MUL
  <ADD
    <INT 2>
    <INT 3>
  >
  <FUNCALL
    <ID f>
    <LIST
      <ADD
        <INT 4>
        <INT 5>
      >
      <NIL>
    >
  >
>
